app/src/main/res/values/strings.xml

```diff
@@ -18,6 +19,9 @@
   <string name="option_should_store_cookies">shouldStoreCookies</string>
   <string name="option_set_http_connect_timeout">setHttpConnectTimeout</string>
   <string name="option_set_http_read_timeout">setHttpReadTimeout</string>
+  <string name="documentation_should_store_cookies">Set if http client should store cookies between requests. The default value is TRUE.</string>
+  <string name="documentation_set_http_connect_timeout">Set the http client connect timeout. If set to a value &lt;= 0, 60000 (60 seconds) is used.</string>
```

Comment:
Set the http client connect timeout in seconds. Minimum value needs to be higher than 0,6 (60 milliseconds).

Reply:
It's an integer value in milliseconds. It can be anything above the 0, but if we set 0 or lower, then the default 60000ms is used. I updated the text to make it clearer what are the units here